### PR TITLE
COPY FROM support for CSV, DELIMITER, NULL

### PR DESCRIPTION
**Pull request by ericharmeling**
Fixes https://github.com/cockroachdb/docs/issues/10061.
Fixes https://github.com/cockroachdb/docs/issues/10062.

### CODE REVIEW

File: v21.1/copy-from.md [hunk 1]
```diff
@@ -12,23 +12,26 @@ CockroachDB currently only supports `COPY FROM` statements issued from third-par
...
 
 ### Options
 
 Option | Description
 -----------|-------------
-`BINARY` | Copy data from binary format.<br>If not specified, CockroachDB copies in plaintext format. Note that CSV format is not yet supported.
+`DELIMITER 'value'` | <span class="version-tag">New in v21.1:</span> The value that delimits the rows of input data, passed as a string.
+`NULL 'value'` | <span class="version-tag">New in v21.1:</span> The string that represents a `NULL` value in the input data.
+`BINARY` | Copy data from binary format. If `BINARY` is specified, no other format can be specified.<br>If no format is specified, CockroachDB copies in plaintext format.
+`CSV` | <span class="version-tag">New in v21.1:</span> Copy data from CSV format. If `CVS` is specified, no other format can be specified.<br>If no format is specified, CockroachDB copies in plaintext format.
```
Comment (rafiss) [inline v21.1/copy-from.md:34]: typo on the third "CSV"

**Review by rafiss — CHANGES_REQUESTED** [review]
lgtm but one typo!

File: v21.1/copy-from.md (hunk 1 above)
Comment (ericharmeling) [reply v21.1/copy-from.md]: Good catch. Fixed.

**Comment by ericharmeling** [comment]
TFTR @rafiss !

**Review by ianjevans — APPROVED** [review]

**Pull request merged**